nidn/fdtd_integration/compute_spectrum_fdtd.py

```diff
@@ -153,3 +170,28 @@ def _average_along_detector(signal):
             s[2] += p[2] / len(signal[i])
         avg[i] = s
     return avg
+
+
+def _summed_thickness_times_permittivity(thicknesses, permittivity):
+    summed_permittivity = 0
+    for i in range(len(thicknesses)):
+        summed_permittivity += thicknesses[i] * permittivity[:, :, i].real
+    return summed_permittivity
+
+
+def _check_sufficient_timesteps(
+    cfg: DotMap, thicknesses, wavelength, permittivity, scaling
+):
+    recommended_timesteps = int(((
```

Comment:
why not compute the scaling and thicknesses here where you use them? :) makes more sense codewise I think

Reply:
I see, will do:)